readme.md

```diff
@@ -1,114 +1,23 @@
-Download Shortcut Sensei from the [Chrome Web Store](https://chrome.google.com/webstore/detail/shortcut-sensei/ahhliekflgdfbgofmgamanngngnbaibh/related) or the [Firefox Add-Ons Store](https://addons.mozilla.org/en-US/firefox/addon/shortcutsensei/)
+<p align="center">
+<img src="https://user-images.githubusercontent.com/73364073/141783737-79275213-6e18-4553-808f-3503456ca5ff.png" width="300"/>
```

Comment:
@LennardVaarten it's best practice to commit these images to the version control repository (so that anybody can change it) and make the address relative (../image.png) instead of absolute (https://...png) 

See an example in former line 6 below.

Reply:
@LennardVaarten I went ahead and fixed this here: https://github.com/janmechtel/shortcutsensei/commit/824ece6788af02c7d8fe99e2db81673a6620035c